lxpanel-plugin/kano_profile.c

```diff
@@ -53,6 +59,29 @@ static void menu_pos(GtkMenu *menu, gint *x, gint *y, gboolean *push_in,
                      GtkWidget *widget);
 static gboolean profile_status(kano_profile_plugin_t*);
 static void plugin_destructor(gpointer user_data);
+void file_monitor_cb(GFileMonitor *monitor, GFile *first, GFile *second,
+             GFileMonitorEvent event, gpointer user_data);
+
+gchar *get_profile_filename(void);
+
+
+gchar *get_profile_filename(void)
+{
+    struct passwd *pw = getpwuid(getuid());
+    const char *homedir = pw->pw_dir;
+
+    // You are responsible for freeing the returned char buffer
```

Comment:
@alex5imon Maybe this warning should be more prevalent in a function comment rather than just inline so that someone just looking for the function finds this immediately.